sonar-project.properties

```diff
@@ -6,7 +6,7 @@ sonar.tests=src/test/
+sonar.exclusions=src/main/steps/applicant/confirm-contact-details/checkanswers/controller/ApplicantConfirmContactDetailsPostController.ts,src/main/steps/applicant/confirm-contact-details/checkanswers/controller/ApplicantConfirmContactDetailsGetController.ts,src/main/app/document/DocumentManagementController.ts,src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsPostController.ts,src/main/app/controller/GetCaseController.ts,src/main/app/controller/PostController.ts,src/main/app/controller/RespondentSubmitResponseController.ts,src/main/steps/common/confirm-contact-details/checkanswers/ContactDetailsMapper.ts,src/main/steps/common/confirm-contact-details/checkanswers/ConfirmContactDetailsGetController.ts,src/main/steps/tasklistresponse/start/tasklist.ts,src/main/steps/tasklistresponse/tasklistresponseCaseSequence.ts,src/main/app/case/DgsApiClient.ts,src/main/app/document/DocumentDetail.ts,src/main/app/document/DeleteDocumentRequest.ts,src/main/app/document/DocumentManagementClient.ts,src/main/app/document/GenerateAndUploadDocumentRequest.ts,src/main/steps/respondent/task-list/tasklist.ts,**/__mocks__/**,src/**/*content.ts,src/**/definition.ts,src/**/steps/constants.ts,src/**/willingness-to-attend-miam/miam-cost-exemptions.ts,src/**/steps/urls.ts,src/**/case/CaseApi.ts,src/main/steps/respondent/task-list/utils.ts,src/main/steps/c100-rebuild/confirmation-page/ApplicationDownloadController.ts,src/main/app/case/C100CaseApi.ts
```

Comment:
Hi @hmcts-abduldaim , can you please remove this as well from sonar file and add test cases for it. Please make sure that there are no c100-rebuild files in sonar
src/main/steps/c100-rebuild/confirmation-page/ApplicationDownloadController.ts